lib/dry/component/container.rb

```diff
@@ -34,6 +34,10 @@ def self.configure(env = config.env, &block)
         self
       end
 
+      def self.Loader(key)
```

Comment:
It's a quick hack to be able to change the `Loader` function on a [subclass](https://github.com/sagmor/dry-component/blob/namespace/lib/dry/component/namespace.rb#L35).
I did it like this to touch as little as possible the current container and get the proof of concept working but definitely should be changed.